### PR TITLE
New empty sub item state for folders

### PR DESCRIPTION
**Pull request by ehconitin**
#8809 
@Bonapara 
Figuring out animations :) WIP

Screenshot 2024-12-06 at 5 28 35 PM

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerEmptySubItem.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+const StyledItemLabel = styled.span`
+  font-weight: ${({ theme }) => theme.font.weight.medium};
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`;
+
+const StyledNavigationDrawerItemContainer = styled.span`
+  display: flex;
+  width: 100%;
+`;
```
Comment (martmull) [inline packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerEmptySubItem.tsx:50]: duplicates with `<NavigationDrawerItem`

Comment (ehconitin) [reply packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerEmptySubItem.tsx:50]: it is! I opted for a separate component.
Or I could add a prop isEmptyState. Should I?

**Review by martmull — CHANGES_REQUESTED** [review]
Hey, it seems that there is a lot of duplication between `<NavigationDrawerEmptySubItem` and `<NavigationDrawerItem />`

I think we should not create a new component `<NavigationDrawerEmptySubItem`, but adapt `<NavigationDrawerItem />` so it fit the design

**Comment by charlesBochet** [comment]
@ehconitin Discussed with Thomas that said this is not needed anymore, as this PR is also staled, I'm closing it. Feel free to re-open it if it still makes sense

**Pull request closed without merging**